src/components/VirtualThought.tsx

```diff
@@ -38,6 +39,10 @@ export type OnResize = (args: {
 /** Selects the cursor. */
 const selectCursor = (state: State) => state.cursor
 
+/** Selects whether the context view is active for this thought. */
+const selectShowContexts = (path: SimplePath) => (state: State) =>
+  equalPath(state.cursor, path) && isContextViewActive(state, path)
```

Comment:
Is `equalPath(state.cursor, path)` necessary? I think just selecting the context view state should work.

Reply:
Good catch, omitting. 👍